bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -679,6 +679,33 @@ impl Connection {
         Ok(self)
     }
 
+    /// Retrieves the size of the session ticket.
+    pub fn session_ticket_length(&mut self) -> Result<usize, Error> {
```

Comment:
Should this be &mut self or just &self? Is this mutating?

Same for session_ticket

Reply:
with tls1.3 tickets arriving in potentially other tasks, I set it to mut self to ensure a valid access and avoid any data races.